src/module/effect/SR5ActiveEffect.ts

```diff
@@ -287,14 +287,14 @@ export class SR5ActiveEffect extends ActiveEffect {
      */
     static applyOverrideToModifiableValue(effect: SR5ActiveEffect, model: DataModel.Any, change: ActiveEffect.ChangeData, current, delta) {
         const modValue = SR5ActiveEffect.getModifiableValue(model, change.key);
-        if (modValue) {
-            modValue.override = { name: effect.name, value: Number(change.value) };
-            modValue.value = Number(change.value);
+        if (!modValue) return false;
 
-            return true;
-        }
+        const value = Number(change.value);
+        if (isNaN(value)) return true;

```

Comment:
This isNaN could be relevant for the string variant of ModifiableValue.

While I don't see an application for strings on `upgrade` and `downgrade` modes, `override` very much would work for strings.

I think, this here is allright and the string variant should likely be it's own PR for simplicity. :)